docs/book/releases.md

```diff
@@ -0,0 +1,57 @@
+<!-- markdownlint-disable MD033 -->
+<!-- markdownlint-disable MD004 -->
+<!-- markdownlint-disable MD034 -->
+# vSphere CSI Driver Releases
+
+- [v2.1.0](#v2.1.0)
+- [v2.0.1](#v2.0.1)
+- [v2.0.0](#v2.0.0)
+- [v1.0.2](#v1.0.2)
+- [v1.0.1](#v1.0.1)
+
+## v2.1.0<a id="v2.1.0"></a>
+
+* Refer https://github.com/kubernetes-sigs/vsphere-csi-driver/releases/tag/v2.1.0 for features listing and notable changes.
+* Known issues with v2.1.0 release
+  * `v2.1.0` has issue with static file volume provisioning. When the file PV is re-created with the same PV name, volume is not getting registered as a container volume with vSphere and thus further operations like attach/delete can not be performed on the volume.
+  * `v2.1.0` has a bug in the metadata-syncer container which can delete the volume physically upon PV deletion even when PV is marked with reclaim policy `retain`.
```

Comment:
Can you explain briefly how this could happen or are the details in some other doc?  Is there a workaround?

Reply:
@xing-yang I have corrected the statement here. you can re-read this as 

>   * `v2.1.0` has a bug in the metadata syncer container which may delete the volume physically from the datastore when the Bound PV with reclaim policy delete is deleted by the user when `StorageObjectInUseProtection` is disabled on Kubernetes Cluster.

workaround is to keep `StorageObjectInUseProtection` enabled. but to protect from accidental data loss we should prevent deleting volume physically from syncer container. 

Referring to the code here - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/v2.1.0/pkg/syncer/metadatasyncer.go#L1041-L1044